vm/os-windows.cpp

```diff
@@ -304,6 +304,61 @@ void handle_ctrl_c() {
   SetConsoleCtrlHandler(factor::ctrl_handler, TRUE);
 }
 
+static volatile bool stop_on_ctrl_break = false;
+static HANDLE ctrl_break_thread = nullptr;
+
```

Comment:
I don't know. Because C++11?
typedef void *HANDLE;
http://stackoverflow.com/questions/1282295/what-exactly-is-nullptr